src/components/header/README.md

```
@@ -286,0 +1,253 @@
+const isArticle = isArticlePage(url);
+const isCategory = isCategoryPage(url);
+
+// ✅ Nouveau système (recommandé)
+const detection = pageDetectionManager.detectPage(url);
+const isArticle = detection.pageType === "article";
+const isCategory = detection.pageType === "category";
+```
+
+## 🔥 Avantages de la Nouvelle Architecture
+
+### ✅ **Extensibilité**
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)